src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -16,6 +16,8 @@
 import java.util.HashMap;

```

Comment:
this file shouldn't have any changes in this PR.. perhaps we need to merge develop back into this branch?

Reply:
missed import or ctrl-O'd it out in another commit :shrug: